### DISCUSSION TITLE
BAYCKRC NANO PRO

### DISCUSSION THREAD

**Discussion by Lashtabeg** [ExpressLRS/ExpressLRS#2899] [Ideas]
Could you please add the firmware for BAYCKRC 900Mhz NANO PRO RX.
Thanks

**Comment by pkendall64** [comment]
It's already been added to the configurator. It has been available since firmware release 3.3.1

**Comment by kapel93** [comment]
Could you write firmware name of bayckrc 900mhz nano pro
